registry/remote/repository.go

```diff
@@ -288,57 +324,69 @@ func (r *Repository) tags(ctx context.Context, last string, fn func(tags []strin
 	return parseLink(resp)
 }
 
-// Predecessors returns the descriptors of ORAS Artifact manifests directly
+// Predecessors returns the descriptors of image or artifact manifests directly
 // referencing the given manifest descriptor.
-// Predecessors internally leverages Referrers, and converts the result ORAS
-// Artifact descriptors to OCI descriptors.
-// Reference: https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md
+// Predecessors internally leverages Referrers.
+// Reference: https://github.com/opencontainers/distribution-spec/blob/main/spec.md#listing-referrers
```

Comment:
Let's use a tagged version instead of `main`. Please also apply the update to other docs.